src/Bicep.Core.Samples/Files/Parameters_LF/main.bicepparam.json

```diff
@@ -0,0 +1,65 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
```

Comment:
I manually added this. How do I update the baseline script to do it for me?

Reply:
See https://github.com/Azure/bicep/pull/7498#discussion_r915027028